#An array of pipe line segments which represent a length of pipe
geometry_msgs/Pose2D initial_pose
int64 id
bool sub_pipe_line
int64 connect_id
float64 connect_distance
float64 theta
PipeLineSegment[] pipe_line
float64 diameter
float64 pipe_thickness



